tests/artist-delete.test.js

```diff
@@ -0,0 +1,32 @@
+const { expect } = require('chai')
+const request = require('supertest')
+const db = require('../src/db')
+const app = require('../src/app')
+
+describe('Delete Artist', () => {
```

Comment:
Your indentation needs fixing in this whole file, this describe block covers the entire rest of the file so everything inside it should be indented